.github/workflows/CI.yml

```diff
@@ -463,3 +463,24 @@ jobs:
             gitSha=${{ env.GIT_SHA }}
           push: ${{ matrix.config.should-push-image == 'true' && (( github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository ) || ( github.event_name == 'push' )) && github.actor != 'renovate[bot]' && github.actor != 'dependabot[bot]' }}
           pull: true
+
+  calculate-total-runtime:
+    name: End-of-Pipeline Metrics
+    if: always()
```

Comment:
This step will probably fail for PRs from forks - should check that in the if.

Reply:
aha! good catch, i will fix that

Reply:
done